modularity/Modularity.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>for weighted networks, <code>G.to_undirected()</code> won't do what we want (see https://networkx.org/documentation/stable/reference/classes/generated/networkx.DiGraph.to_undirected.html)</p><p> </p><p>Please update this so that when you force a directed graph to be undirected, you take the average i -&gt; j and j -&gt; i edge weight.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neurodata/connectocross/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I tried to fix that, please tell me if it's still not something that you want